contracts/extensions/treasury.clar

```diff
@@ -2,7 +2,7 @@
 ;; SPDX-License-Identifier: BUSL-1.1
 
 (use-trait proxy-trait .proxy-trait.proxy-trait)
-(use-trait sip-010-transferable-trait .sip-010-transferable-trait.sip-010-transferable-trait)
+(use-trait sip-010-transferable-trait .sip-010-transferable-trait-v1-01.sip-010-transferable-trait)
```

Comment:
In contrast to what we discussed, it would make more sense to use the SIP10 "default" sip trait.

Reply:
This is for meta protocols to better deal with contracts that use sip-010-traits.

Reply:
Done.